code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm

```diff
@@ -0,0 +1,258 @@
+#define STATUS_EFFECT_BLAZING /datum/status_effect/blazing
+#define STATUS_EFFECT_BLINDED /datum/status_effect/blinded
+//CREDIT TO REDACTED FOR HELPING ME WITH THIS UNHOLY CODE
+/mob/living/simple_animal/hostile/abnormality/fire_bird
+	name = "The Firebird"
+	desc = "A large bird covered in ashes, pray its feathers do not re-ignite."
+	icon = 'ModularTegustation/Teguicons/96x96.dmi'
+	icon_state = "firebird_inert"
+	icon_living = "firebird_active"
+	threat_level = WAW_LEVEL
+	maxHealth = 2000
+	health = 2000
+	max_boxes = 24
+	pixel_x = -32
+	base_pixel_x = -32
+	pixel_y = -16
+	base_pixel_y = -16
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(55, 55, 50, 50, 60),
+		ABNORMALITY_WORK_INSIGHT = list(30, 30, 25, 25, 35),
+		ABNORMALITY_WORK_ATTACHMENT = list(45, 45, 40, 40, 50),
+		ABNORMALITY_WORK_REPRESSION = list(45, 45, 40, 40, 50)
+		)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+	faction = list("hostile")
+	can_breach = TRUE
+	start_qliphoth = 3
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.4, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 2.0)
+	light_color = COLOR_LIGHT_ORANGE
+	light_range = 0
+	light_power = 0
+	loot = list(/obj/item/gun/ego_gun/feather)
+	ego_list = list(/datum/ego_datum/armor/feather)
+	gift_type = /datum/ego_gifts/feather
```

Comment:
Change these variables to something more fitting. Right now it nuzzles you.
While cute, it's not actually fitting
friendly_verb_continuous = ""
friendly_verb_simple = ""

Reply:
I gave the the verb "grazes" for now, as itd probably pass by people, does that work?